--- /tmp/OpenThreads/include/OpenThreads/Atomic
+++ /tmp/OpenThreads-svn-13083/include/OpenThreads/Atomic
@@ -241,7 +241,7 @@
 AtomicPtr::assign(void* ptrNew, const void* const ptrOld)
 {
 #if defined(_OPENTHREADS_ATOMIC_USE_GCC_BUILTINS)
-    return __sync_bool_compare_and_swap(&_ptr, (void *)ptrOld, ptrNew);
+    return __sync_bool_compare_and_swap(&_ptr, ptrOld, ptrNew);
 #elif defined(_OPENTHREADS_ATOMIC_USE_MIPOSPRO_BUILTINS)
     return __compare_and_swap((unsigned long*)&_ptr, (unsigned long)ptrOld, (unsigned long)ptrNew);
 #elif defined(_OPENTHREADS_ATOMIC_USE_SUN)
