From 534d88a9f977d8ad68b199fe35555b973e45f71a Mon Sep 17 00:00:00 2001
From: "[CST] Focus" <focus.cst@gmail.com>
Date: Mon, 6 Oct 2014 15:57:26 +0400
Subject: [PATCH 9/9] libavformat/utils.c: fix linking issue that breaks
 ffprobe

---
 libavformat/utils.c | 20 ++++++++++++++++++++
 1 file changed, 20 insertions(+)

diff --git a/libavformat/utils.c b/libavformat/utils.c
index e095d60..926b400 100644
--- a/libavformat/utils.c
+++ b/libavformat/utils.c
@@ -110,6 +110,26 @@ MAKE_ACCESSORS(AVFormatContext, format, int, metadata_header_padding)
 MAKE_ACCESSORS(AVFormatContext, format, void *, opaque)
 MAKE_ACCESSORS(AVFormatContext, format, av_format_control_message, control_message_cb)
 
+void *av_fast_realloc(void *ptr, unsigned int *size, size_t min_size)
+{
+    if (min_size < *size)
+        return ptr;
+
+    min_size = FFMAX(17 * min_size / 16 + 32, min_size);
+
+    ptr = av_realloc(ptr, min_size);
+    /* we could set this to the unmodified min_size but this is safer
+     * if the user lost the ptr and uses NULL now
+     */
+    if (!ptr)
+        min_size = 0;
+
+    *size = min_size;
+
+    return ptr;
+}
+
+
 int64_t av_stream_get_end_pts(const AVStream *st)
 {
     return st->pts.val;
-- 
2.1.1

